lib/nostrum/cache/guild/guild_server.ex

```diff
@@ -386,7 +386,7 @@ defmodule Nostrum.Cache.Guild.GuildServer do
   end
 
   def handle_call({:update, :member, guild_id, new_partial_member}, _from, state) do
-    old_member = state.members[new_partial_member.user.id]
+    old_member = Map.get(state.members, new_partial_member.user.id, %{})
```

Comment:
For the default here we should supply an empty member struct to better adhere to the documentation for the consumers.